api/organisations/management/commands/create_licensing_private_key.py

```diff
@@ -0,0 +1,10 @@
+from typing import Any
+
+from django.core.management import BaseCommand
+
+from organisations.subscriptions.licensing.helpers import create_private_key
+
+
+class Command(BaseCommand):
+    def handle(self, *args: Any, **options: Any) -> None:
+        print(create_private_key())
```

Comment:
```suggestion
        self.stdout.write(create_private_key())
```

We should use `self.stdout` here instead of print as per advice in django docs (see [here](https://docs.djangoproject.com/en/5.1/howto/custom-management-commands/))

Reply:
(this comment applies to all usages of print below)

Reply:
Interesting. Thanks!